src/SurveyService/Controllers/UserAnswerController.cs

```diff
@@ -0,0 +1,21 @@
+﻿using LT.DigitalOffice.Kernel.Responses;
+using LT.DigitalOffice.SurveyService.Business.Commands.UserAnswer.Interfaces;
+using LT.DigitalOffice.SurveyService.Models.Dto.Requests.UserAnswer;
+using Microsoft.AspNetCore.Mvc;
+using System;
+using System.Threading.Tasks;
+
+namespace LT.DigitalOffice.SurveyService.Controllers;
+
+[Route("[controller]")]
+[ApiController]
+public class UserAnswerController : ControllerBase
+{
+  [HttpPost("create")]
+  public async Task<OperationResultResponse<Guid?>> CreateAsync(
```

Comment:
request - List<OptionId> (if question can have several answers (hasMultipleChoice=true) or it is a group of questions, receive answers one by one has no sense; if it is a single question with one answer there would be a List just of one)
response OperationResultResponse<bool> - Id here is technical, don't need to return it

Reply:
Done